code/lib/cli/src/initiate.ts

```diff
@@ -319,9 +320,29 @@ async function doInitiate(options: CommandOptions, pkg: PackageJson): Promise<vo
 
   logger.log('\nFor more information visit:', chalk.cyan('https://storybook.js.org'));
 
-  if (projectType === ProjectType.ANGULAR) {
+  const isReactProject =
+    projectType === ProjectType.REACT_SCRIPTS ||
+    projectType === ProjectType.REACT ||
+    projectType === ProjectType.WEBPACK_REACT ||
+    projectType === ProjectType.REACT_PROJECT ||
+    projectType === ProjectType.NEXTJS;
+
+  const shouldRunDev =
+    projectType !== ProjectType.ANGULAR && projectType !== ProjectType.REACT_NATIVE;
+  if (process.env.CI !== 'true' && shouldRunDev) {
+    logger.log('\nRunning Storybook');
+    await dev({
+      ...options,
+      port: 6006,
+      open: true,
+      quiet: true,
+      // TODO: change this logic to all frameworks once the idea is validated
+      initialPath: isReactProject ? '/onboarding' : undefined,
+    });
+  } else if (projectType === ProjectType.ANGULAR) {
     logger.log('\nTo run your Storybook, type:\n');
     codeLog([`ng run ${installResult.projectName}:storybook`]);
```

Comment:
Can/should we automate this command for angular projects also?

Reply:
Sure thing!